test/Makefile

```diff
@@ -37,9 +39,10 @@ endif
 .PHONY: conditional_bear_make
 conditional_bear_make:
 ifeq ($(BEAR_INSTALLED),yes)
-	@echo "Bear is installed, running with bear..."
+	echo "Bear is installed, running with bear..."
 	bear $(MAKE) -C $(BUILD_MAKEFILE_DIR) all ; \
-	mv compile_commands.json ../.vscode/compile_commands.json
+	mkdir -p $(VS_CODE_DIR) ; \
+	if [ -f compile_commands.json ]; then mv compile_commands.json $(VS_CODE_DIR)/compile_commands.json; fi
```

Comment:
i believe there was a change I made to this that made it work with older versions of bear on main but i may not remember correctly. can u just merge in main

Reply:
Idk what you mean. As of speaking, this is already up to date with main